<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputTesting</class>
 <widget class="QWidget" name="OutputTesting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(27, 150, 208);
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>261</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>主伺服电机测试</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#ffffff;&quot;&gt;速    度&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus_Out_pb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>41</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string> + </string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus_Out_lb">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>41</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pointmove_Out_pb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>点 动 (F6)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_Out_lb">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>停 止(F7)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="speed_lb">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>5</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plusandminus_lb">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>91</width>
      <height>35</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>6</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>251</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>信号灯测试</string>
   </property>
   <widget class="QPushButton" name="needlelight_pb">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>235</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="toolTip">
     <string>4</string>
    </property>
    <property name="text">
     <string>关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yellowlight_pb">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>3</string>
    </property>
    <property name="text">
     <string>关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="greenlight_pb">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>125</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="toolTip">
     <string>2</string>
    </property>
    <property name="text">
     <string>关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="redlight_pb">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="toolTip">
     <string>1</string>
    </property>
    <property name="text">
     <string>关</string>
    </property>
   </widget>
   <widget class="QLabel" name="r_lb">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>21</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>|</string>
    </property>
   </widget>
   <widget class="QLabel" name="g_lb">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>125</y>
      <width>21</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="h_lb">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>21</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="n_lb">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>21</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="red_lb">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>红色</string>
    </property>
   </widget>
   <widget class="QPushButton" name="green_lb">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>125</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>绿色</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yellow_lb">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>黄色</string>
    </property>
   </widget>
   <widget class="QPushButton" name="needle_lb">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>针筒照明灯</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="mesg_lb">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>注：按F5进行区域切换，左右方向键进行加减速</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>50</y>
     <width>241</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
font-size:20px;
}
QPushButton{
background-color:transparent;
color:white;
font-size:20px;
border:none;
font:bold;
}</string>
   </property>
   <property name="title">
    <string>橡筋张力测试</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2路橡筋张力+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2路橡筋张力-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>6路橡筋张力+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>6路橡筋张力-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1-3-5-7张力+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1-3-5-7张力-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2-4-6-8张力+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2-4-6-8张力-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xiangjing_1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="toolTip">
     <string>1</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:2px solid green;
font:bold;font-size:18px;
color:red;
background-color: rgb(255, 255, 127);
}
QPushButton::checked{
color:green;
}</string>
    </property>
    <property name="text">
     <string>关</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="xiangjing_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="toolTip">
     <string>1</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:2px solid green;
font:bold;font-size:18px;
color:red;
background-color: rgb(255, 255, 127);
}
QPushButton::checked{
color:green;
}</string>
    </property>
    <property name="text">
     <string>关</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="xiangjing_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="toolTip">
     <string>1</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:2px solid green;
font:bold;font-size:18px;
color:red;
background-color: rgb(255, 255, 127);
}
QPushButton::checked{
color:green;
}</string>
    </property>
    <property name="text">
     <string>关</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="xiangjing_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="toolTip">
     <string>1</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:2px solid green;
font:bold;font-size:18px;
color:red;
background-color: rgb(255, 255, 127);
}
QPushButton::checked{
color:green;
}</string>
    </property>
    <property name="text">
     <string>关</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="xiangjing_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="toolTip">
     <string>1</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:2px solid green;
font:bold;font-size:18px;
color:red;
background-color: rgb(255, 255, 127);
}
QPushButton::checked{
color:green;
}</string>
    </property>
    <property name="text">
     <string>关</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="xiangjing_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="toolTip">
     <string>1</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:2px solid green;
font:bold;font-size:18px;
color:red;
background-color: rgb(255, 255, 127);
}
QPushButton::checked{
color:green;
}</string>
    </property>
    <property name="text">
     <string>关</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="xiangjing_7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="toolTip">
     <string>1</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:2px solid green;
font:bold;font-size:18px;
color:red;
background-color: rgb(255, 255, 127);
}
QPushButton::checked{
color:green;
}</string>
    </property>
    <property name="text">
     <string>关</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="xiangjing_8">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="toolTip">
     <string>1</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:2px solid green;
font:bold;font-size:18px;
color:red;
background-color: rgb(255, 255, 127);
}
QPushButton::checked{
color:green;
}</string>
    </property>
    <property name="text">
     <string>关</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>420</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:10;
background-color: rgb(80, 80, 80);
font-size:24px;
color:rgb(255, 255, 0);
}
QPushButton::pressed{
background-color: rgb(120, 120, 120);
}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
